tests/e2e/exampleE2E.test.js

```diff
@@ -32,3 +32,33 @@ describe('preference-setting test', () => {
     expect(text2['_remoteObject'].value).toBe("SAVED");
   });
 });
+
+describe('recipe-searchPage test', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://127.0.0.1:5500/source/recipe-searchPage.html');
```

Comment:
I will figure out a way to let firebase export some env variable for this file to pick up in the future. For now, this is good since we are only running it on local.

Reply:
Oh for sure, that's my bad. I will add new files to add those tests later.